proxy/default.conf.template

```diff
@@ -168,3 +168,12 @@ server {
         proxy_pass http://$erpnext;
     }
 }
+
+server {
+    listen 8083;
+    location / {
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+        proxy_set_header X-Forwarded-Proto $scheme;
+        proxy_pass http://fhir-odoo:8080;
```

Comment:
hey @enyachoke @corneliouzbett 
IMO, Nginx proxy is failing to start for us since we don't run oddo.  
```
2024/10/16 08:26:52 [emerg] 1#1: host not found in upstream "fhir-odoo" in /etc/nginx/conf.d/default.conf:177
nginx: [emerg] host not found in upstream "fhir-odoo" in /etc/nginx/conf.d/default.conf:177
``` 
Maybe, we can use a variable for the resolution to be done at runtime rather than startup

Reply:
That's true.
Let me fix that.